lib/uplinkc/uplink_definitions.h

```diff
@@ -51,6 +51,27 @@ typedef struct EncryptionParameters {
     int32_t     block_size;
 } EncryptionParameters;
 
+
+typedef struct EncryptionRestriction {
+    char *bucket;
+    char *path_prefix;
+} EncryptionRestriction;
+
+typedef struct CaveatPath {
+	char *bucket;
+	char *encrypted_path_prefix;
+} CaveatPath;
+
+typedef struct Caveat { 
+	bool disallow_reads;
+	bool disallow_writes;
+	bool disallow_lists;
+	bool disallow_deletes;
+	//ToDo: CaveatPath *allowed_paths;
```

Comment:
remove those TODOs because it brings no value at the moment.

Reply:
TODOs removed